Testing System/src/main/java/com/epam/practice/testingsystem/data/dao/DbAccess.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.practice.testingsystem.data.dao;
+
+import com.epam.practice.testingsystem.data.DataConnection;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+abstract class DbAccess {
+    protected static final String dbAccessExceptionMessage = "An error occurred while trying to access the database";
+
+    protected static int getLastInsertedId(Connection connection) throws SQLException {
+        try (PreparedStatement statement = connection.prepareStatement("SELECT LAST_INSERT_ID()")) {
+            ResultSet rs = statement.executeQuery();
+            rs.next();
+            return rs.getInt(1);
+        }
+    }
+
+    protected void delete1Arg(String table, String field, int id) {
+        String sql = String.format("DELETE FROM %s WHERE %s = ?", table, field);
```

Comment:
Please add the method for checking the inputs to avoid SQL injection.

The example of threat can be found here: https://www.w3schools.com/sql/sql_injection.asp